tests/components/fyta/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,39 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'config_entry': dict({
+      'data': dict({
+        'access_token': '123xyz',
```

Comment:
Isn't an access token just as bad to leave unexposed?

Reply:
Good point. Will add this to the redacted items.